### ISSUE TITLE
is it possible to train from scratch?

### ISSUE THREAD

**Issue by leadcain84** [leeyoshinari/YOLO_v2#2]
Thanks you for source code.

In the loss function, warm-up method is not implemented.

are you try to train from the scratch without pre-trained weight?

**Comment by leeyoshinari** [comment]
I tried to train from scratch, but the precision of model was very low. If you train well, we can exchange experiences.
